packages/View/src/index.js.flow

```diff
@@ -45,7 +45,7 @@ declare module "constelation-view" {
     paddingHorizontal?: number | string,
     pointerEvents?: string,
     position?: string,
-    refNode?: () => {},
+    refNode?: (node?: Base) => void,
```

Comment:
This will actually return a React Native `View` in all 3 cases. So I don't think we'll need the refNode overrides below.